samples/npm1300_fuel_gauge/README.rst

```diff
@@ -0,0 +1,90 @@
+.. _npm1300_fuel_gauge:
+
+nPM1300 fuel gauge
+##################
+
+This sample calculates battery state of charge information using the :ref:`lib_nrf_fuel_gauge` library.
+
+Requirements
+************
+
+The sample support the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an NPM1300-EK PMIC.
+
+Overview
+********
+
+This sample is a reference for calculating state of charge, time to empty and time to full information
+from a battery connected to an NPM1300 PMIC.
+
+Wiring
+******
+
+Connect the TWI interface between the development kit and the NPM1300-EK
+
```

Comment:
For tables, see https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/cheat_sheet.html#tables for help.